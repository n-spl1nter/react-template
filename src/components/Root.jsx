import React from 'react';

const Root = () => {
  return(
    <div>
      <h1 style={{textAlign: 'center'}}>Root Component</h1>
    </div>
  );
};

export default Root;
