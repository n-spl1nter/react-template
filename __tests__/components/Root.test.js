import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Root from '../../src/components/Root';

describe('Root component tests', () => {
  const rootComponent = shallow(<Root />);
  it('should correct snapshot render',  () => {
    expect(shallowToJson(rootComponent)).toMatchSnapshot();
  });
});
