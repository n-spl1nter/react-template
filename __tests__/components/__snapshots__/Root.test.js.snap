// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root component tests should correct snapshot render 1`] = `
<div>
  <h1
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Root Component
  </h1>
</div>
`;
